src/guide/forms.md

```diff
@@ -1,5 +1,7 @@
 # Form Input Bindings
 
+<VideoLesson href="https://vueschool.io/lessons/user-inputs-vue-devtools-in-vue-3?friend=vuejs" title="Learn how to handle form inputs on Vue School">Learn how to work with form inputs with a free Vue School lesson</VideoLesson>
```

Comment:
While it may be misleading, at this point, users don't know about components yet. So instead of diving into that rabbit hole, our intention is to excite the viewer

Reply:
I think it is more likely to frustrate them when they waste hours trying to find documentation for a feature that doesn't exist.

Reply:
agreed. I can cut out that sentence.